### ISSUE TITLE
Storage of PDF -- edit contents

### ISSUE THREAD

**Issue by RebeccaGreiner** [amosproj/amos2024ss05-knowledge-graph-extractor#3]
## User story
1. As a system user,
2. I want the parsed documents to be stored in an object storage,
3. so that the textual information can be used as the basis for constructing the knowledge graph.

## Acceptance criteria
* Parsed documents are automatically transferred to the object storage upon successful parsing.
* Storage should be done so that later, after the LLM runs, the result and each knowledge node can be linked to the original text corpurses (title/subtitle/page number) so the documentation can be found.
* Storage should be done in a way so that if a customer-facing UI needs to be added in after, it is easily expandable.
* The storage architecture is scalable to accommodate an increasing volume of parsed documents.

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
